src/scss/abstracts/_variables.scss

```diff
@@ -10,5 +10,4 @@ $green-dark: #043326;
 $grey: #d8d8d8;
 $orange: #ffcc36;
 $yellow: #ffcc36;
-$white: #ffffff;
-
+$white: #fff;
```

Comment:
please add an empty line to the end of the file

Reply:
Done!